Rakefile

```diff
@@ -52,3 +52,10 @@ task preservation_errors: [:environment] do
   error_reporter = ErrorReporter.new
   error_reporter.output
 end
+
+desc 'retrieve preservation logs'
```

Comment:
perhaps a better description would be something like 'get druid counts for each WF step' ???  (or 'retrieve')

and for error reporter:  'get error and warning messages for each WF step' ???